third_party/move/move-vm/runtime/src/loader/function.rs

```diff
@@ -535,6 +541,14 @@ impl Function {
         &self.param_tys
     }
 
+    pub fn is_persistent(&self) -> bool {
+        self.is_persistent || !self.is_friend_or_private()
```

Comment:
Should `entry` functions also be considered persistent in this context?

Reply:
From the language viewpoint, I would actually prefer to use `#[persistent] entry fun` for this, since its a feature flag enabled at a certain point in the Aptos past.